tasties_app/models.py

```diff
@@ -46,6 +55,65 @@ class UnitChoices(models.TextChoices, Enum):
     )
     description = models.CharField(max_length=64, validators=[MinLengthValidator(1)])
 
+    def __str__(self):
+        return str(self.amount) + " " + self.measurement_unit + " " + self.description
+
+    @staticmethod
+    def is_enum_value(value):
+        """
+        Checks if the given value is a valid UnitChoices enum value.
+        """
+        for unit in Ingredient.UnitChoices:
+            if unit == value:
+                return True
+        return False
+
+    @staticmethod
+    def add(recipe, amount, measurement_unit, description):
```

Comment:
Could you please explain why you need this method? Why not just use the `save` method?

Reply:
+1. 
Django takes care of the validators you defined in the model class by himself,
so I agree with Nati here that this add function can be omitted.